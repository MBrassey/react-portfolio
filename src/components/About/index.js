import ProfilePicture from "../../assets/cover/profile.jpg";
import React from "react";

function About() {
  return (
    <section className="my-5">
      <div className="my-2">
        <div className="my-5">
          <img
            src={ProfilePicture}
            alt="Matt's Profile Picture"
            style={{
              borderRadius: "3%",
              overflow: "visible",
              boxShadow: "0 12px 24px 0 rgba(0, 0, 0, 0.7)",
              height: "240px",
              width: "240px",
              marginTop: "9px",
              border: "solid 3px black",
            }}
          />
        </div>
        <div className="about-text">Welcome, I'm Matt Brassey - Full stack blockchain developer with SysOps & DevOps background. Known among staff for strong wit and attention to detail no matter the complexity of the project or severity of itâ€™s operations. Proven experience applying continuous integration within small group and large, corporate team settings. Strong ability to modularize large, complex project objectives into concise and manageable parts. Recently completed a 24-week bootcamp earning a certificate in full stack development from the University of Arizona. Intense interest in distributed systems, decentralized finance and blockchain technology. Keen technologist with an insatiable desire to learn, build, improve and collaborate on world changing platforms and applications.</div>
      </div>
    </section>
  );
}

export default About;
