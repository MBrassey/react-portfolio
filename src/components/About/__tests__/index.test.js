import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import About from '..';

afterEach(cleanup);

describe('About component', () => {
  it('renders', () => {
    render(<About />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<About />);

    expect(asFragment()).toMatchSnapshot();
  });
})
