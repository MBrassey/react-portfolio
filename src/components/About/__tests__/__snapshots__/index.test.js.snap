// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      style="transition: opacity 400ms, transform 400ms; transform: translateY(20px); opacity: 0;"
    >
      <section
        class="my-5"
      >
        <div
          class="my-2"
        >
          <div
            class="my-5 profileImg"
          >
            <img
              alt="Matt's Profile Picture"
              src="profile.jpg"
              style="border-radius: 3%; overflow: visible; box-shadow: 0 12px 24px 0 rgba(0, 0, 0, 0.7); height: 100%; width: 100%; margin-top: 9px; border: 3px solid black;"
            />
          </div>
          <div
            class="about-text"
          >
            I'm 
            <strong>
              Matt Brassey
            </strong>
             - Full stack blockchain developer with SysOps & DevOps background. Known among staff for strong wit and attention to detail no matter the complexity of the project or severity of itâ€™s operations. Proven experience applying continuous integration within small group and large, corporate team settings. Strong ability to modularize large, complex project objectives into concise and manageable parts. Recently completed a 24-week bootcamp earning a certificate in full stack development from the University of Arizona. Intense interest in distributed systems, decentralized finance and blockchain technology. Keen technologist with an insatiable desire to learn, build, improve and collaborate on world changing platforms and applications.
          </div>
        </div>
      </section>
    </div>
  </div>
</DocumentFragment>
`;
